runtime/src/main.rs

```diff
@@ -51,6 +52,16 @@ pub extern "C" fn entry_point() -> ! {
     };
     caliptra_common::stop_wdt(&mut drivers.soc_ifc);
 
+    if !cfg!(feature = "no-cfi") {
```

Comment:
Following up on this in chat. I think there's a decent chance the linker will optimize this out to remove the if statement. But just want to be sure, since glitching past CFI initialization could cause some odd behavior.

Reply:
Confirmed